requestPairingCode() cannot have function default parameters.

### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Is this a problem caused by your code, or is it specifically because of the library?

- [X] I have double-checked my code carefully.

### Describe the bug.

After build my node.js app with `tsc`, when trying to start the server running `node dist/src/index.js`, the following error is throw:

```
node_modules/whatsapp-web.js/index.d.ts:127:49 - error TS2371: A parameter initializer is only allowed in a function or constructor implementation.
requestPairingCode(phoneNumber: string, showNotification = true): Promise<string>
```

### Expected Behavior

`requestPairingCode` method shoud be correcly declared.
Because abstract functions cannot have function default parameters.

✅ Correct way:
`requestPairingCode(phoneNumber: string, showNotification: boolean): Promise<string>`

❌ Incorrect way:
`requestPairingCode(phoneNumber: string, showNotification = true): Promise<string>`

### Steps to Reproduce the Bug or Issue

Run:
1. `npm install whatsapp-web.js@v1.26.1-alpha.2`
2. `tsc`
3. `node dist/src/index.js`

### WhatsApp Account Type

Standard

### Browser Type

Chromium

### Operation System Type

Windows

### Phone OS Type

iOS

### WhatsApp-Web.js Version

1.26.1-alpha.2

### WhatsApp Web Version

2.3000.1017054665

### Node.js Version

22.0.0

### Authentication Strategy

LocalAuth

### Additional Context

I fix this typo error updating the method in my `node_modules` folder, changing this:
`requestPairingCode(phoneNumber: string, showNotification = true): Promise<string>`

To this:
`requestPairingCode(phoneNumber: string, showNotification: boolean): Promise<string>`

I got the same issue while building my app, **here's the error details**:
```tsx
> server@1.0.0 build
> npm run clean && tsc


> server@1.0.0 clean
> rm -rf dist

node_modules/whatsapp-web.js/index.d.ts:127:49 - error TS2371: A parameter initializer is only allowed in a function or constructor implementation.

127         requestPairingCode(phoneNumber: string, showNotification = true): Promise<string>
                                                    ~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/whatsapp-web.js/index.d.ts:127



> server@1.0.0 clean
> rm -rf dist0 build
> npm run clean && tsc
node_modules/whatsapp-web.js/index.d.ts:127:49 - error TS2371: A parameter initializer is only allowed in a function or constructor implementation.

127         requestPairingCode(phoneNumber: string, showNotification = true): Promise<string>
                                                    ~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/whatsapp-web.js/index.d.ts:127
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -160,7 +160,7 @@ declare namespace WAWebJS {
          * @param showNotification - Show notification to pair on phone number
          * @returns {Promise<string>} - Returns a pairing code in format "ABCDEFGH"
          */
-        requestPairingCode(phoneNumber: string, showNotification = true): Promise<string>
+        requestPairingCode(phoneNumber: string, showNotification?: boolean): Promise<string>
 
         /** Force reset of connection state for the client */
         resetState(): Promise<void>

